test/lib/search.bats

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bats
+
+load ../../lib/composure
+load ../../plugins/available/base.plugin
+
+cite _about _param _example _group _author _version
+
+load ../../lib/helpers
+load ../../lib/search
+
+NO_COLOR=true
+
+@test "helpers search aliases" {
+  run _bash-it-search-component 'plugins' 'base'
+  [[ "${lines[0]}" =~ 'plugins' && "${lines[0]}" =~ 'base' ]]
+}
+
+@test "helpers search all ruby et al" {
```

Comment:
It would be nice to have an additional test for the `-chruby` case as well. Can you add another test case for this, please?